pepper-apis/ddp-common/pom.xml

```diff
@@ -12,6 +12,19 @@
     <artifactId>ddp-common</artifactId>
 
     <dependencies>
+        <!-- unclear why we need to put logback here explicitly vs. inheriting it, but without it dsm's SlackAppender.java won't
```

Comment:
Curious if anyone knows how to deal with this more elgantly.

Reply:
probably because of L151

Reply:
Wait, which line?  I only see 89 lines in this file.